tensorflow/c/kernels.h

```diff
@@ -184,6 +185,10 @@ TF_CAPI_EXPORT extern void TF_OpKernelConstruction_GetAttrInt32(
     TF_OpKernelConstruction* ctx, const char* attr_name, int32_t* val,
     TF_Status* status);
 
+// Returns the name of the user-defined NodeDef for this OpKernel. 
```

Comment:
NodeDef is an implementation detail (there will be a way to get it without node def in the future). May be "Returns name of the operations (potentially user-defined)."
In some cases it is not user defined if user just passes None. Then TF just assigns a unique name.

Reply:
Out of curiosity, could the phrasing "returns the name of the operation" be confused with the name of the opdef? The name of the nodedef could be different from the opdef.

Reply:
Indeed that could be confusing. I am going off what we say in Python docs: https://www.tensorflow.org/api_docs/python/tf/math/abs
Also here:
https://source.corp.google.com/piper///depot/google3/third_party/tensorflow/core/framework/node_def.proto;l=17
Also, I really don't want to mention `NodeDef` since with TFRT Kernel Fallback work, we will implement this API without any NodeDef behind it.

May be "name of the operation instance" or "unique operation name".

Reply:
Gotcha, that sounds good!

Reply:
I changed to "returns name of the unique operation name."